samples/snippets/powershell/VS_Snippets_CLR_System/ps/structure1.ps1

```diff
@@ -0,0 +1,40 @@
+# <snippet1>
```

Comment:
Since there's only one example here, do you need the snippet tags?

Reply:
as far as the snippet tags - I thought these were needed to match up with the "#1" in the XML.

Reply:
@doctordns, use snippet tags if you have multiple examples in a single file. If the example is the entire file, don't use snippet tags in the file, and don't include the snippet reference after the filename and file path.

Reply:
Right. I've remove the snippet tag, removed that weird additioin to XML and I think put the snippet in the right place. Have pushed this all up and let's see what happens.

I will eventually get this all figured out at my end! ;-)